tests/register/register.go

```diff
@@ -110,6 +115,27 @@ func (test Test) UnmountPartitions(t *testing.T, loopDevice string) {
 func (test Test) RunCoreOSInstall(t *testing.T, loopDevice string, opts ...string) {
 	opts = append(opts, "-d", loopDevice)
 
+	// use local image if it's available and we aren't testing any version picks directly
```

Comment:
This will mean that we're testing `-f` most of the time, which is the opposite of what users will do, and won't exercise the fetching and signature checking code as much as we could.  I'd rather have explicit tests for `-f`, and default to overriding `-b` instead (and serving HTTP locally).